distribution/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM python:3.12.1-slim-bullseye
+
+WORKDIR /app
+
+COPY ../requirements /tmp/requirements/
+
+RUN pip install -r /tmp/requirements/base.txt
+
+COPY .. .
```

лучше указывать название папки и точный конечный путь
```suggestion
COPY /src/product /src/product
```